components/nofrendo/libsnss/libsnss.c

```diff
@@ -182,7 +182,7 @@ SNSS_WriteFileHeader (SNSS_FILE *snssFile)
    char writeBuffer[8];
 
    /* always place the SNSS tag in this field */
-   strncpy (&writeBuffer[0], "SNSS", 4);
+   strncpy (&writeBuffer[0], "SNSS", 5);
```

Comment:
That's an interesting change. The original was correct actually, but I imagine the compiler complained about this.